### ISSUE TITLE
[Adaptive Cards - Inputs with Validation] -Ensure links are distinguished from surrounding text in a way that does not rely on color.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8800]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 23H2 (OS Build 26020.1000)
Browser Version: Version 122.0.2348.0 (Official build) dev (64-bit)
Tool: Accessibility Insight for Web

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Navigate to 'Activity Update' option and press enter key to activate it.
4. Run the accessibility insight for web tool and observe the issue.

**Actual Result:**
Links are not distinguished from surrounding text in a way that does not rely on color.

**Expected Result:**
Ensure links should distinguished from surrounding text in a way that does not rely on color.

**Element path:**
.w3-col.m6.w3-margin-top > p > a[target="_blank"][rel="noopener"]

**Snippet:**
a target="_blank" rel="noopener" href="https://docs.microsoft.com/adaptive-cards/templating/">Get started with templating <i class="fas fa-chevron-right"></i></a

**How to fix:**
Fix any of the following:
  The link has insufficient color contrast of 2.97:1 with the surrounding text. (Minimum contrast is 3:1, link text: #0050c7, surrounding text: #000000)
  The link has no styling (such as underline) to distinguish it from the surrounding text

**Note:**
Same issue is repro on the 'Action.Show Card' screen.
Same issue is repro under 'Action.ToggleVisibility' screen.

**User Impact:**
Users with color blindness and low vision have difficulty in identifying the links present on the page. Users can't be able to differentiate between the normal text and links if links do not have underline/chevron symbols.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

“ Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section”

### Screenshots

![WCAG1 4 1 - Ensure links are distinguished from surrounding text in a way that does not rely on color](https://github.com/microsoft/AdaptiveCards/assets/95913748/9e55b960-f674-40e0-a628-9993a92b0c97)

### Card JSON

```JSON
NA
```
